services/frontend/pkg/config/defaults/defaultconfig.go

```diff
@@ -31,7 +31,7 @@ func DefaultConfig() *config.Config {
 			Protocol:  "tcp",
 			Prefix:    "",
 			CORS: config.CORS{
-				AllowedOrigins: []string{"*"},
+				AllowedOrigins: []string{"https://localhost:9200"},
```

Comment:
why not use the shared config value for the ocis url as a default?

Reply:
Because it was also hardcoded in line 82 https://github.com/owncloud/ocis/blob/0d52f89d1ffc61345674c436ac619a2002f96633/services/frontend/pkg/config/defaults/defaultconfig.go#L82